src/tests.cpp

```diff
@@ -0,0 +1,85 @@
+#include "quadratic_equation.h"
+
+struct CoefficentsForLinear {
+    double first;
+    double second;
+    double root;
+    int return_roots;
+};
+
+struct CoefficentsForQuadratic {
+    double first;
+    double second;
+    double third;
+    double root_first;
+    double root_second;
+    int return_roots;
+};
+
+bool TestIsZero(double num[3], bool check_veriable) {

```

Comment:
1) Magic number 3; unit test is supposed to print what it tests, what result is correct and what was actually returned. Something like
```
[OK] IsZero(0) returned true; expected true.
[INCORRECT] IsZero(220202) returned true; expected false.
```
2) TYPO AGAIN!!!
3) No need to pass an array for testing. You can take one argument, but call TestIsZero in a loop. But if you do pass an array, at least don't return from the function when an error is encountered, you should test all values!